core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1420,6 +1420,17 @@ class KafkaConfig(val props: java.util.Map[_, _], doLog: Boolean, dynamicConfigO
   override def get(key: String): AnyRef =
     if (this eq currentConfig) super.get(key) else currentConfig.get(key)
 
+  override def postProcessParsedConfig(props: java.util.Map[String,Object]): java.util.Map[String,Object] = {
```

Comment:
Why are we doing the validation in this method since it always returns an empty map? Can we do the same validation in `validateValues`? `KafkaConfig` performs the more complicated validation in that method.

Reply:
Good point!  I'll move it.